cmd/ssh.go

```diff
@@ -233,10 +344,31 @@ func (cmd *SSHCmd) forwardPorts(ctx context.Context, containerClient *ssh.Client
 	return <-errChan
 }
 
-func (cmd *SSHCmd) startTunnel(ctx context.Context, devPodConfig *config.Config, containerClient *ssh.Client, ideName string, log log.Logger) error {
+func (cmd *SSHCmd) startTunnel(
```

Comment:
in general those kind of refactorings I'd put in separate PR - there is already a lot of lines of code :)